### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by lozalojo**

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"I18N.LANGUAGE": "Español",
+	"I18N.MAINTAINERS": "@Viriato139ac#0342",
```
Comment (bekriebel) [inline lang/es.json]: Small thing, but can you please replace the tabs at the beginning of these lines with four spaces so it is consistent with the rest of the lines?

**Review by bekriebel — CHANGES_REQUESTED** [review]
Thanks for the submission! Just one small change requested.

File: lang/es.json (hunk 1 above)
Comment (lozalojo) [reply lang/es.json]: Done!

**Review by bekriebel — APPROVED** [review]

**Comment by bekriebel** [comment]
@lozalojo I recently added some new strings to the module. I'd love for the translations to be updated as well if you have some spare time for them. :-) Thanks again for this work!

**Comment by lozalojo** [comment]
> @lozalojo I recently added some new strings to the module. I'd love for the translations to be updated as well if you have some spare time for them. :-) Thanks again for this work!

Sure, I'll take a look

**Pull request merged**